examples/recv_sd/recv_sd.ino

```diff
@@ -31,6 +31,9 @@ char str[20];
 void setup()
 {
     SERIAL.begin(115200);
+    while (!Serial) {
```

Comment:
Should be 
```
while (!SERIAL) {
```

Reply:
Ok. But where is the difference to what i wrote?
Please have a look at the example code [here](https://www.arduino.cc/reference/en/language/functions/communication/serial/ifserial/)

Reply:
```
/*SAMD core*/
#ifdef ARDUINO_SAMD_VARIANT_COMPLIANCE
  #define SERIAL SerialUSB
#else
  #define SERIAL Serial
#endif
```
SERIAL means can be serial console, different platforms are not the same

Reply:
Now i see it. You are right. But the programm should wait for the serial console to be available. (Example: SAMD-variant right after uploading the code). So this line should stay `while (!Serial) {` but the line above is supposed to be `Serial.begin(115200);`